templates/bundles/FOSUserBundle/Security/login.html.twig

```diff
@@ -1,6 +1,7 @@
 {% extends '@AdminLTE/FOSUserBundle/Security/login.html.twig' %}
 
-{% block logo_login %}<b>Kimai</b><br>TimeTracking{% endblock %}
+{% block logo_login %}{% include 'partials/logo_login.html.twig' %}{% endblock %}
```

Comment:
@lduer Is there a reason for the difference between `logo_login.html.twig` and `login_logo.html.twig`?
Was that by intention and a file is missing or is it a bug? See comment: https://github.com/tobybatch/kimai2/issues/5#issuecomment-456747207

Reply:
Clarified, see #520

Reply:
Thanks for fixing :) obviously a typo